src/site/content/en/blog/how-mercadolibre-optimized-web-vitals/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: How Mercado Libre optimized for Web Vitals (TBT/FID)
+subhead: Optimizing interactivity of Mercado Libre's product details pages.
+authors:
+  - demianrenzulli
+  - aranhacarlos
+  - joanbaca
+date: 2020-09-01
+hero: hero.jpg
+description: Summary of the work done by Mercado Libre's frontend architecture team to optimize FID, using TBT as a proxy metric.
+tags:
+  - blog
+  - performance
+  - web-vitals
+---
+
+Mercado Libre is the largest e-commerce and payments ecosystem in Latin America. It is present in 18 countries and is a market leader in Brazil, Mexico, and Argentina (based on unique visitors and pageviews).
+
+Web performance has been a focus for the company for a long time. They recently formed a team to monitor performance and apply optimizations across different parts of the site.
+
+This article summarizes the work done by [Guille Paz](https://twitter.com/pazguille), [Pablo Carminatti](https://www.linkedin.com/in/pcarminatti/), and [Oleh Burkhay](https://twitter.com/oburkhay) from Mercado Libre's frontend architecture team to optimize one of the Core Web Vitals: [First Input Delay (FID)](https://web.dev/fid/) and it's lab proxy, [Total Blocking Time (TBT)](https://web.dev/tbt/).
+
```

Comment:
Can you add a [Stats component](https://web.dev/handbook/web-dev-components/#stats) here that summarizes the main successes?

Reply:
I've added two stats, please let me know if they look good.

Reply:
LGTM